quickstart-node/README.md

```diff
@@ -26,6 +30,7 @@ npm i
 ```
 
 ## Compiling Files
```

Comment:
Is the compiling and running sections true? is this how you've done it?

Reply:
Yes, this is the script property in the package.json:
`"scripts": {
    "build": "tsc",
    "start": "node -r ts-node/register/transpile-only -r tsconfig-paths/register dist/index.js",
    "dev": "nodemon",
    "fetch:schema": "bash fetch-schema.sh",
    "codegen": "graphql-codegen",
    "fetch:generate": "npm run fetch:schema && npm run codegen"
  },`